.github/workflows/build_windows.yaml

```diff
@@ -0,0 +1,56 @@
+name: Build standalone tool for Windows
+
+on: 
+  push:
+    branches:
+      - master
```

Comment:
I would say it could be triggered by push to any branch, not just master...

Reply:
fixed